CONTRIBUTING.md

```diff
@@ -147,17 +147,17 @@ pnpm run --filter @rsbuild/some-package test
 
```

Comment:
This PR contains several changes, can you split them to into separate PRs? 😄

Small PR is better for reviewing and generating changelog.

Reply:
Sure. I'm closing this PR for now.